### PR TITLE
excessive MTU logs/v6

### PR DESCRIPTION
**Pull request by inashivb**
Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket: https://redmine.openinfosecfoundation.org/issues/5831

Previous PR: https://github.com/OISF/suricata/pull/9400

Changes since v5:
- in debug validation checks, first check if iface is null and then its items

### CODE REVIEW

**Comment by suricata-qa** [comment]
WARNING:
 
 | field | baseline | test | % |
|--|--|--|--| 
| | SURI_TLPW2_autofp_stats_chk | | |
| .flow.spare | 1951025 | 2163493 | 110.89% | 

Pipeline 15714

File: src/util-ioctl.c [hunk 1]
```diff
@@ -55,26 +55,16 @@
  *
  * \param Name of a network interface
  */
-static int GetIfaceMaxHWHeaderLength(const char *pcap_dev)
+static int GetIfaceMaxHWHeaderLength(const char *dev)
```
Comment (catenacyber) [inline src/util-ioctl.c:58]: Why this renaming ?

File: src/util-ioctl.c [hunk 2]
```diff
@@ -114,11 +114,15 @@ int GetIfaceMTU(const char *dev)
  * for the link. In case of uncertainty, it will output a
  * majorant to be sure avoid the cost of dynamic allocation.
  *
- * \param Name of a network interface
+ * \param LiveDevice object
  * \retval 0 in case of error
  */
-int GetIfaceMaxPacketSize(const char *dev)
+int GetIfaceMaxPacketSize(LiveDevice *ld)
```
Comment (catenacyber) [inline src/util-ioctl.c:120]: Why do we need this change ? (to put in the commit message I guess)

File: src/util-ioctl.c (hunk 1 above)
Comment (inashivb) [reply src/util-ioctl.c:58]: Renaming was done following this review comment: https://github.com/OISF/suricata/pull/9372#discussion_r1290352129

File: src/util-ioctl.c (hunk 2 above)
Comment (inashivb) [reply src/util-ioctl.c:120]: ok. Putting in commit message

**Pull request closed without merging**